css/style.css

```
@@ -15,5 +15,147 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+ 
+ nav  {
+    display:flex;
+    justify-content: flex-end;
+     padding-top: 60px;
+     padding-left: 20px;
+     
+ } 
 
+.nav{
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 50%;
```

```suggestion
    width: 100%;;
```
`top` and `left` only have an effect when we've used the `position` CSS property, but since we haven't used it then we can remove `top` and `left` as they're not doing anything.

`height: 50%` also doesn't have an effect so we should remove it. We usually remove any code that doesn't have any effect